### PR TITLE
Automation Test

### PR DESCRIPTION
**Pull request by labkey-sweta**
#### Rationale
 Reproducibility Report Enhancements automation test
1. Reproducibility Report test case
2. Calibration curve test case
3. Clib File test case.

#### Related Pull Requests
* 

#### Changes
additional test cases in test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java

### CODE REVIEW

File: test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java [hunk 1]
```diff
@@ -93,6 +114,75 @@ public void testSingleReplicateFile()
...
+        importData(SKY_FILE_FOR_CALIBRATION_CURVE);
+
+        clickTab("Proteins");
+
+        checker().verifyFalse("Reproducibility report link should not be present", isElementPresent(reproducibilityReportLink()));
+        checker().verifyTrue("Calibration curve icon should be present", isElementPresent(calibrationCurvesLink()));
+
+        clickAndWait(calibrationCurvesLink().index(2));
+
+        log("Verifying the FOM values");
```
Comment (labkey-jeckels) [inline test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java:134]: If we're just clicking on the third link (and not identifying it based on the protein name) let's also verify that we're getting to the peptide we're expecting, along with the upper and lower limits

File: test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java [hunk 2]
```diff
@@ -93,6 +114,75 @@ public void testSingleReplicateFile()
...
+        checker().verifyEquals("Incorrect Lower limit of quantitation", "1.0",
```
Comment (labkey-jeckels) [inline test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java:137]: Might need to wait for these values - they're loaded via AJAX so they may lag the initial page load

**Review by labkey-jeckels — APPROVED** [review]
A couple of suggestions but no need for an additional review unless you'd like one.

File: test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java (hunk 1 above)
Comment (labkey-sweta) [reply test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java:134]: In the Protein tab, Protein/Label column  as it listed as TG|1578-1589|z2 upon clicking reproducibility report link it navigates to page with title "Calibration Curve: VIFDANAPVAVR"

Verify for sequence "VIFDANAPVAVR". ?

Comment (labkey-jeckels) [reply test/src/org/labkey/test/tests/targetedms/TargetedMSMxNReproducibilityReportTest.java:134]: Yes, sounds good.

**Pull request merged**